pkg/features/features.go

```diff
@@ -35,4 +35,9 @@ const (
 	// owner @ggaurav10, @amshuman-kr
 	// alpha: v0.1.0
 	HVPA utilfeature.Feature = "HVPA"
+
+	// HVPAForShootedSeed enables simultaneous horizontal and vertical in Soil Clusters.
```

Comment:
```suggestion
	// HVPAForShootedSeed enables simultaneous horizontal and vertical scaling in Soil Clusters.
```

Reply:
Don't call it "soil", that's no official term. Use "shooted seed".